src/parsers/pm2ps_parser.jl

```diff
@@ -263,41 +263,76 @@ function make_thermal_gen(gen_name, d, bus)
     return thermal_gen
 end
 
-function read_gen(data,Buses)
+"""
+Transfer generators to ps_dict according to their classification
+"""
+function read_gen(data, Buses, genmap_file=nothing)
+
+    if genmap_file == nothing
+        genmap_file = joinpath(dirname(dirname(pathof(PowerSystems))),
+                               "src/parsers/generator_mapping.toml")
+    end
+    genmap_dict = TOML.parsefile(genmap_file) # not working... map to current path??
```

Comment:
Is this also a stale comment?

Reply:
Yep! I guess need to review my code a little better before submitting...